receiver/hostmetricsreceiver/internal/scraper.go

```diff
@@ -38,9 +38,6 @@ type Scraper interface {
 
 // Factory can create a Scraper.
 type Factory interface {
-	// Type gets the type of the scraper created by this factory.
-	Type() string
-
```

Comment:
Was this not removed?

Reply:
Yes I just needed to do add this here to make this branch since I had both PRs open at the same time. I just rebased and this is gone now